src/app/pages/wallets/my-wallets/pages-wallets-my-wallets.service.ts

```diff
@@ -16,4 +16,16 @@ export class PagesWalletsMyWalletsService {
       map(walletsResp => walletsResp.map(resp => new MyWallet(resp)))
     );
   }
+
+  public addNewWallet(name_: string): Observable<MyWallet> {
```

Comment:
Names here were not fixed. Still have `_`. And I think you can just pass here `wallet: MyWallet` instead only name. And for update the same.
You can extend `MyWallet` class so `id` can be `number | null` (`null` for newly created wallets that were not saved yet in database).
Those classes can also have `toPayload` method that will return payload (`IWalletPayload`) used in the gateway.